packages/typir/src/kinds/class-kind.ts

```diff
@@ -63,7 +305,7 @@ export class ClassKind implements Kind {
     readonly options: ClassKindOptions;
```

Comment:
Class kind feels like a collection of utility functions and not as something with its own purpose/responsibility. (Markus+Insa during Typir meeting)

Reply:
I am not happy with the current design as well: Sometimes, kinds need to store some information (e.g. FunctionKind), therefore, stateless utility functions are not enough.

Reply:
I will leave the current design for now, but we need to discuss and revise it in the future again!